preview_generator/preview/builder_factory.py

```diff
@@ -71,6 +71,10 @@ def get_file_mimetype(self, file_path: str, file_ext: str = "") -> str:
             ).communicate()[0]
             str_ = raw_mime.decode("utf-8").replace("\n", "")
 
+        if not str_:
+            # Should never happen.
```

Comment:
`mime.from_file` should never return `None`, but the lib is not typed, so mypy is lost here. We're ensuring ourself that `str_` is not `None` to disambiguate the situation.